ert3/storage/_storage.py

```diff
@@ -426,32 +461,89 @@ def add_ensemble_record(
     dataframe = pd.DataFrame([r.data for r in ensemble_record.records])
     record_type = _get_record_type(ensemble_record)
 
-    parameters = _get_experiment_parameters(workspace, experiment_name)
-    if record_name in parameters:
-        # Split by columns
-        for column_label in dataframe:
+    if record_type == ert.data.RecordType.LIST_BYTES:
+        _add_blob_data(
+            experiment_name, record_name, ensemble_record, ensemble_size=ensemble_size
+        )
+    else:
+        parameters = _get_experiment_parameters(workspace, experiment_name)
+        if record_name in parameters:
+            # Split by columns
+            for column_label in dataframe:
+                _add_numerical_data(
+                    workspace,
+                    experiment_name,
+                    f"{record_name}.{column_label}",
+                    dataframe[column_label],
+                    record_type,
+                )
+        else:
             _add_numerical_data(
                 workspace,
                 experiment_name,
-                f"{record_name}.{column_label}",
-                dataframe[column_label],
+                record_name,
+                dataframe,
                 record_type,
             )
+
+
+def _add_blob_data(
```

Comment:
Would it be possible to merge this function and `_add_numerical_data` simular with how `_get_numerical_data` became `_get_data`

Reply:
Will revisit this when we get a BlobRecord